### DISCUSSION TITLE
Voxit users: please add your name!

### DISCUSSION THREAD

**Discussion by millerlm99** [MillerLab-UCDavis/Vocal_Analysis_Tools#1] [General]
## Please, PLEASE let us know if you are using Voxit for your research -- this will help us continue the project! Just add your name and affiliation in this discussion, with any notes about your research topic if you choose. Thank you so much -- Lee and Marit

NAME
AFFILIATION  
RESEARCH TOPIC/USE (optional)

Prof. Lee M. Miller and Prof. Marit Macarthur
University of California, Davis
analyzing poet voices, etc

**Comment by wilfle22** [comment]
Isabelle Stuart and William Fleming
University of Oxford; University of Cambridge
Poet voices

**Comment by PoeticaSonoraMX** [comment]
The PoéticaSonoraMX [research group members](https://poeticasonora.unam.mx/acerca-de/) 
Coordinated by Prof. Susana González Aktories and Aurelio Meza, PhD
Universidad Nacional Autónoma de México; Concordia University
Poet voice, Spoken Word, and Literary Audio in Spanish and English

**Comment by dcfaltesek** [comment]
Dan Faltesek - Oregon State
Public Address, Presidential Speeches

**Comment by kwsherwood** [comment]
Ken Sherwood
Indiana University of Pennsylvania
Modernist poetry and digital humanities

**Comment by dcfaltesek** [comment]
Dan Faltesek Oregon state, new media.
